.travis.yml

```diff
@@ -6,8 +6,12 @@ cache:
   - cocoapods
 
 stages:
-  - checks
-  - test
+  - name: checks
```

Comment:
Doesn't seem to be working. I don't see a build on travis?

Reply:
I think, I filtered out `checks` stage. I'll put it back.